src/components/Header/ProfileIcon.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from 'react'
+
+function ProfileIcon () {
+    
+  const [contact, setContact] = useState({})
+
+  const userName = "TomEastwood"
+  const baseUrl = `https://boolean-api-server.fly.dev/${userName}`
```

Comment:
I'd consider moving `https://boolean-api-server.fly.dev/` to its own constant that you can import where needed since it's used a few times. Sometimes I'll make a `constants.js` file for these